### DISCUSSION TITLE
Is it possible to change sky for all phases?

### DISCUSSION THREAD

**Discussion by Magiciandseq3** [nonamecrackers2/crackers-wither-storm-mod#1996] [Q&A] [answered]
Is it possible to change the sky color and fog color for all phases with the resource pack or can we make the sky dark in all phases with another method?

**Comment by nonamecrackers2** [comment, marked as answer]
You cannot currently have the sky darken for all the phases

**Comment by Magiciandseq3** [comment]
it will be added in 4.2?
